ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -406,7 +406,21 @@
     @if (!Model.PuzzleState.IsEmailOnlyMode)
     {
         <h3>Need some help?</h3>
-        <p>While solving this puzzle, if you've run into a bug, think there's an error, or are not having fun anymore, <a asp-Page="/Threads/PuzzleThread" asp-route-puzzleid="@Model.Puzzle.ID" asp-route-teamId="@Model.Team?.ID" asp-route-playerId="@Model.LoggedInUser.ID">use this link to ask for help!</a></p>
+        @if (!Model.Event.HideHints)
```

Comment:
This logic order doesn't match the embedded version (search for the second instance of `<div class="inlinewrapper">`) and probably should.

Reply:
Thanks for the catch. Fixed